### PR TITLE
change player animController && deleteRigidbody

### PR DESCRIPTION
**Pull request by AlenaKarp**

### CODE REVIEW

File: Assets/Art/Prototype/JKT_Art/Animals/cat/ani/idle.controller [hunk 1]
```diff
@@ -1,5 +1,27 @@
 %YAML 1.1
 %TAG !u! tag:unity3d.com,2011:
+--- !u!1101 &-87607477109181869
+AnimatorStateTransition:
+  m_ObjectHideFlags: 1
```
Comment (MrDoctor-666) [inline Assets/Art/Prototype/JKT_Art/Animals/cat/ani/idle.controller:5]: nice

**Review by MrDoctor-666 — DISMISSED** [review]
really nice

**Review by MrDoctor-666 — APPROVED** [review]
yes cool I approve

**Comment by Ku6epnec** [comment]
We aprove this!

**Pull request merged**